run/run.py

```diff
@@ -345,6 +358,28 @@ def get_config():
     return config['default']
 
 
+def patch_wpt(config, platform):
+    '''Applies util/wpt.patch to WPT.
+
+    The patch is necessary to keep WPT running on long runs.
+    jeffcarp has a PR out with this patch:
+    https://github.com/w3c/web-platform-tests/pull/5774
+    '''
```

Comment:
Nit: double-quotes are more common for docstrings, but it's not really important.